Azure/sonic-buildimage
# [as5835-54x] Fix read thermal sensor issue

#### Why I did it
Fix as5835-54x read thermal sensor issue.
Due to linux create hwmonN node number, N isn't fixed. So modify to use "*" to let thermal.py can correct sysfs. For example, use /sys/bus/i2c/devices/18-004b/hwmon/hwmon*/temp1_input to instead of /sys/bus/i2c/devices/18-004b/hwmon/hwmon3/temp1_input
#### How I did it
Fix as5835-54x thermalutil.py.
#### How to verify it
Reboot DUT, and check as5835-54x-platform-monitor.service.
systemctl status as5835-54x-platform-monitor.service -l
● as5835-54x-platform-monitor.service - Accton AS5835-54X Platform Monitoring se
   Loaded: loaded (/lib/systemd/system/as5835-54x-platform-monitor.service; enab
   Active: active (running) since Wed 2021-02-17 03:14:18 UTC; 2h 22min ago

Make sure there is no error message.

Error case that will show below message,
Feb 9 03:13:32.531678 54x-02 INFO accton_as5835_54x_monitor.py[2474]: No such device_path=/sys/bus/i2c/devices/19-004c/hwmon/hwmon4/temp1_input
Feb 9 03:13:32.531831 54x-02 INFO accton_as5835_54x_monitor.py[2474]: No such device_path=/sys/bus/i2c/devices/20-0049/hwmon/hwmon5/temp1_input
......

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
